### ISSUE TITLE
Can't find how to put stress/accent in Greek

### ISSUE THREAD

**Issue by nemlar** [DreymaR/BigBagKbdTrixPKL#11]
Hi!

Firstly thank you very much for all your hard work on this! It really is an amazing tool!

Probably this is not the right spot for this kind of question, but I wasn't sure where to post this...
My issue is, I can't find how to put a stress/accent on a letter when using Greek, as in for example the words: άλφα, Άλφα, έλα, από, Θώθ etc. I guess what is referred in the dead-keys doc as: @0b4 = Acute-Sup   ; ´ - Acute, Gr. Tonos-Oxia, superscripts
Which key is used in the Gr locale that you have provided?

Many thanks!

**Comment by DreymaR** [comment]
Yes, you should be able to write άΆ etc as AltGr+' then letter. If you wish, you could move the Acute/Tonos dead key to a first-level spot, let me know if you decide on a good one! I'm not sure how much people actually write the Tonos these days.

If you want proper katharevousa it'll take some more work on the dead key tables to get there. I started, but the number of accent combinations is massive and nobody asked for it so I didn't take the time and effort. Unfortunately, for dead keys you have to provide a definition for each combination such as dasia+oxia+ypogegrammeni (d,o,y/d,y,o/o,d,y/o,y,d/y,d,o/y,o,d – 6 different ways!). So unfless there's a strict rule for which sequence accents are written in, the task is daunting.
